exercises/02/readme.md

```diff
@@ -215,10 +218,14 @@ With a single command, you've initialized a basic OData service project and with
 
 ## Questions
 
-1. Why is there an focus on "TTM" (time to metadata) - what advantages does that bring?
+1. Why is there an focus on "Contracts First" (As all you need to run a service is a service definition) - what advantages does that bring?
```

Comment:
Don't you like TTM? I quite like it, and it livens people up a bit when we say we just made it up ...

Reply:
I also don't like it too much but the main reason why I changed it was to name it as it's referred to in the official documentation https://cap.cloud.sap/docs/about/#contracts-first

Reply:
good reason